modules/topicsFpdModule.js

```diff
@@ -73,6 +100,54 @@ export function processFpd(config, {global}, {data = topicsData} = {}) {
   });
 }
 
+// Added function addListenerToFetchTopics to listen for topic data retured from iframe loaded for SSPs(bidder)
+function addListenerToFetchTopics() {
+  window.addEventListener('message', (e) => {
+    if (e && e.data) {
+      try {
+        let data = JSON.parse(e.data);
+        if (data && data.segment && !isEmpty(data.segment.topics)) {
+          let {domain, topics, bidder} = data.segment;
+          let segmentData = getTopicsData(domain, topics);
+          let updateStore = [];
+          let bidderFoundInStore = false;
+          let storedSegments = JSON.parse(storage.getDataFromLocalStorage(`${initialTopicName}${bidder}`));
```

Comment:
please use https://github.com/prebid/Prebid.js/blob/e0ef967669ff5d4d67ff695b5326c5540ffb146b/src/utils.js#L1374

Reply:
Using 'safeJSONParse' function to parse the json data.